e2e/commands_test.go

```diff
@@ -48,11 +48,17 @@ func testRenderApp(appPath string, env ...string) func(*testing.T) {
 		dir := fs.NewDir(t, "")
 		defer dir.Remove()
 
+		// Build the App
+		cmd.Command = dockerCli.Command("app", "build", ".", "--folder", filepath.Join(appPath, "my.dockerapp"), "--tag", "a-simple-tag", "--no-resolve-image")
+		res := icmd.RunCmd(cmd).Assert(t, icmd.Success)
+		fmt.Println(res.Combined())
```

Comment:
Should we really print the build output?

Reply:
😱